x-pack/plugins/observability/public/pages/alerts/components/alerts_flyout/alerts_flyout.tsx

```diff
@@ -77,8 +77,8 @@ export function AlertsFlyout({
   }
 
   const ruleId = alertData.fields['kibana.alert.rule.uuid'] ?? null;
-  const linkToRule = ruleId && prepend ? prepend(paths.management.ruleDetails(ruleId)) : null;
-
+  const linkToRule =
+    ruleId && prepend ? prepend(`/app/observability/alerts/rules/${ruleId}`) : null;
```

Comment:
Same here https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/pages/rule_details/config.ts#L22 ?

Reply:
I'll use `prepend(paths.observability.ruleDetails(ruleId))`, so that I need to do the path change only in the paths file.

Ideally we should move these variables in a common place in observability plugin, because they are used by both rules and alerts pages. I will create another ticket to replace all hardcoded values of /app/observability/alerts/ and /app/observability/alerts/rules and this should be covered there.